.connector-store/meta.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "HTTP",
+    "owner": "WSO2",
+    "product": "MI",
+    "category": "Networking",
+    "documentationUrl": "https://mi.docs.wso2.com/en/latest/reference/connectors/http-connector/http-overview/",
```

Comment:
Since this is not a valid url until we release 4.4.0, shall we stick with 4.4.0 doc link and change later?